src/components/outlier-analysis-table/OutlierAnalysisTable.js

```diff
@@ -152,9 +152,9 @@ const OutlierAnalyisTable = ({
                         <TableHeaderColumn className={cssPageStyles.right}>
                             {i18n.t('Max')}
                         </TableHeaderColumn>
-                        {/* <TableHeaderColumn className={cssPageStyles.center}>
+                        <TableHeaderColumn className={cssPageStyles.center}>
                             {i18n.t('Mark')}
```

Comment:
I know this was maybe already-existing, but I think this could use some context if possible - what does "Mark" mean?  "Mark for Follow-up" is better, but maybe too big for a column header... cc @cooper-joe

Reply:
Agreed. To reduce space it could be "Mark follow-up". If that is problematic, "Follow-up" would be better than just "Mark" I think. 
This is dependent on this control being used _only_ for follow-up though. "Mark" might have been chosen because different use-cases are marking it for different reasons, I'm not sure. Looking at documentation, it does seem to be intended for marking as follow-up.

Reply:
Follow-up

Reply:
OK 👍  in that case I'd suggest "Mark follow-up"